core/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
@mhemdan Although I totally agree with distributing the code among different modules, I believe we can categorize code in modules in a more arranged way. I believe it may be more organized to have a module handling the model & presentation code and another one handling the view code.

Check the attached screenshot from this [project](https://github.com/hitherejoe/Bourbon)
![screen shot 2018-09-27 at 3 56 29 pm](https://user-images.githubusercontent.com/8607770/46256566-b2eb8a80-c4ac-11e8-8976-80de305c5026.png)

Reply:
we can add models per project I mean if there are a project needs a tv- wear ...etc we can modify the project correct me if I misunderstood your point